### DISCUSSION TITLE
Addition to Projects using JSON for Modern C++

### DISCUSSION THREAD

**Discussion by Wolf-SO** [nlohmann/json#4379] [General]
As far as I know, Notepad++ uses it, see https://notepad-plus-plus.org/downloads/v8.6.7/ and https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15041

I read a section [*Projects using JSON for Modern C++*](https://github.com/nlohmann/json/blob/develop/README.md#projects-using-json-for-modern-c) in the project's `README.md` but *Notepad++* is not listed there. Do you still maintain this section?

**Comment by nlohmann** [comment]
Thanks for reporting! We put Notepad++ on the list.

---
*This reply was drafted by [Claude Code](https://claude.com/claude-code) on behalf of @nlohmann, as part of a review of open discussions.*
